packages/ember-views/tests/views/view/remove_test.js

```diff
@@ -10,44 +9,8 @@ import viewKeyword from 'ember-htmlbars/keywords/view';
 import { objectAt } from 'ember-runtime/mixins/array';
 
 
-// .......................................................
-// removeChild()
```

Comment:
@rwjblue `removeAllChildren` is from the `LegacyViewSupport` mixin but it uses `removeChild` https://github.com/rwjblue/ember.js/blob/remove-container-view/packages/ember-views/lib/mixins/legacy_view_support.js#L35-L39